virttest/storage.py

```diff
@@ -446,7 +446,7 @@ def clone_image(params, vm_name, image_name, root_dir):
                     logging.info("Clone master image for vms.")
                     process.run(params.get("image_clone_command") %
                                 (m_image_fn, image_fn))
-
+            params["image_name_%s" % vm_name] = vm_image_name
             params["image_name_%s_%s" % (image_name, vm_name)] = vm_image_name
```

Comment:
can we reuse the same param ?

Reply:
Few other tests uses that old param so, I do not want to disturb it for backward compatibility and the newly added param is required to keep the created image mapped to multi vm.